content/1.7/back-office/quick-access/add-edit-quick-access.md

```diff
@@ -7,9 +7,9 @@ weight: 1
 
 The following fields/toggles are displayed:
 
-- **Name**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "Forbidden characters:<>;=#{}". When trying to save and if the field is invalid the following error message is displayed: "The name field is invalid". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. 
- - **URL**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The link field is required.".
-The following tooltip is displayed: "If it's an URL that comes from your back office, you MUST remove the security token.".
+- **Name**: this field is mandatory for the default language. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "The maximum allowed characters is 32.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. The field has a character limit of 32, it does not allow to enter more of them.
+ - **URL**: this field is mandatory. The character limit is 128, it's not possible to enter anymore of them.
+The following tooltip is displayed: "The character limit for this field is 128".
```

Comment:
@Saimis777 @marwachelly

I don't see the tooltip "The character limit for this field is 128"
This is the tooltip displayed on my side:
<img width="1233" alt="Capture d’écran 2022-01-25 à 15 37 19" src="https://user-images.githubusercontent.com/13449658/150997644-2b6f8080-f142-40ac-93ff-0196035d0ef1.png">